### DISCUSSION TITLE
Node-js packages

### DISCUSSION THREAD

**Discussion by asmanp2012** [AliMD/alwatr#115] [Q&A]
How should we have the node-js package in monorepo and manage that?

**Comment by alimd** [comment]
Good question, find the best practices for seperate build

Example: https://github.com/lit/lit/tree/main/packages/labs/ssr

**Reply by asmanp2012** [reply]
I think `lerna run build` .

**Reply by alimd** [reply]
اونکه بله چون هر پکیج ممکنه بیلد های دیگه ای داشته باشه
اما آیا لازمه tsconfig هم جدا بشه یا میشه بگیم مشترک بیلد بشه
فوقش اگر استثنایی بود تو ریفرنس هات tsconfig روت نمیزاری و جدا بیلد میشه

**Reply by alimd** [reply]
قوانین لینت و ... هم ،که قطعا مشترک هستش

**Reply by asmanp2012** [reply]
لیت از tsconfig مشترک استفاده نکرده ولی من توی پروژه هام مشترک در نظر گرفتم مشکلی نداشتم از طرفی هم به نظرم عین جمله ای که گفتی بهترین گزینه هستش
